composer.json

```
@@ -0,0 +1,47 @@
+{
+    "name": "m6web/rate-limit-bundle",
+    "type": "symfony-bundle",
+    "authors": [
+        {
+            "name": "M6Web",
+            "email": "opensource@bedrockstreaming.com",
+            "homepage": "https://tech.bedrockstreaming.com/"
+        }
+    ],
+    "config": {
+        "bin-dir": "bin",
+        "vendor-dir": "vendor",
+        "sort-packages": true
+    },
+    "require": {
+        "php": "7.4.*",
+        "ext-json": "^7.4",
+        "doctrine/annotations": "1.10.*",
```

```suggestion
        "doctrine/annotations": "^1.10.0",
```

we don't want to limit composer requirements too much when we create a bundle
=> we want to allow the bundle users to use `doctrine/annotations 1.11.0` if they want